<launch>
<arg name="ns"/>

<group ns="$(arg ns)">
	<node pkg="joy" name="joy_node" type="joy_node"/>
	<node pkg="px4" name="manual_input" type="manual_input"/>
	<node pkg="px4" name="commander" type="commander"/>
	<node pkg="px4" name="mc_mixer" type="mc_mixer"/>
	<node pkg="px4" name="attitude_estimator" type="attitude_estimator"/>
	<node pkg="px4" name="position_estimator" type="position_estimator"/>
	<node pkg="px4" name="mc_att_control" type="mc_att_control"/>
	<node pkg="px4" name="mc_pos_control" type="mc_pos_control"/>
	<node pkg="px4" name="mavlink" type="mavlink"/>
	<!-- <node pkg="rosbag" type="record" name="record" output="screen" args="-a -O px4_multicopter"/> -->
</group>

</launch>
